packages/app-landing/src/Navigate.tsx

```diff
@@ -0,0 +1,233 @@
+          </div>
+          <div className='action'>
+            <Link to='/accounts'>
+              <button className='button'>Go to Accounts</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faExchangeAlt} color='blue' size='3x' />
+          </div>
+          <div className='header'>Send assets</div>
+          <div className='content'>
+            Send your CENNZnet assets between addresses.
+          </div>
+          <div className='action'>
+            <Link to='/generic-asset'>
+              <button className='button'>Go to Send assets</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faAddressBook} color='blue' size='3x' />
+          </div>
+          <div className='header'>Create an address book</div>
+          <div className='content'>
+            Make sending transactions easier by creating an address book of the
+            accounts that you frequently interact with.
+          </div>
+          <div className='action'>
+            <Link to='/addressbook'>
+              <button className='button'>Go to Address book</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faCompass} color='blue' size='3x' />
+          </div>
+          <div className='header'>Explore network activities</div>
+          <div className='content'>Find out what's happening on CENNZnet.</div>
+          <div className='action'>
+            <Link to='/explorer'>
+              <button className='button'>Go to Explorer</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faTools} color='blue' size='3x' />
+          </div>
+          <div className='header'>Developer tools</div>
+          <div className='content'>
+            Looking for something more advanced? Find all the developer tools here.
+          </div>
+          <div className='action'>
+            <Link to='/contracts'>
+              <button className='button'>Open Advanced</button>
+            </Link>
+          </div>
+        </div>
+        <div className='feature-not-supported'>
+          <div className='icon'>
+            <FontAwesomeIcon icon={faCoins} color='black' size='3x' />
+          </div>
+          <div className='header'>Staking / Delegate</div>
+          <button className='no-action-button'>Coming soon</button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default styled(Navigate)`
+  padding-top: 90px;
+  height: 983px;
+  width: 100%;
+  background-color: #fafafa;
+
+  .explore-featured-fun {
+    margin-left: 79px;
+    height: 58px;
+    width: 477px;
+    color: #5c5d5d;
```

Comment:
We have scoped color defination in a file, and described in this PR - https://github.com/cennznet/apps/pull/17/files

So that whenever design team want to change color theming, we do not need to search `5c5d5d` across the app to replace with new color. And there is other benefits for color defination in file, for example you could aleways use `color: ${colors.red}`, `color: ${colors.brandPrimary}` or `color: ${colors.warning}` in your components without search the exact color code.

Would suggest refactor color implements in landing page with the color config.